sdk-extensions/autoconfigure/src/test/java/io/opentelemetry/sdk/autoconfigure/TracerProviderConfigurationTest.java

```diff
@@ -97,8 +98,7 @@ void configureSpanProcessor_empty() {
                 assertThat(worker)
                     .extracting("queue")
                     .isInstanceOfSatisfying(
-                        ArrayBlockingQueue.class,
-                        queue -> assertThat(queue.remainingCapacity()).isEqualTo(2048));
+                        Queue.class, queue -> assertThat(JcTools.capacity(queue)).isEqualTo(2048));
```

Comment:
The existing logic is verifying the remainingCapacity(). Do you want to do the same?
```
     (JcTools.capacity(queue) - JcTools.size(queue)).isEqualTo(2048)
```

Reply:
Nah, it only used tha tmethod since the JDK only provides that one, but capacity is what we're checking